client/lib/plans/features-list.js

```diff
@@ -757,22 +757,22 @@ export const FEATURES_LIST = {
 	},
 	[ constants.FEATURE_MALWARE_SCANNING_DAILY_AND_ON_DEMAND ]: {
 		getSlug: () => constants.FEATURE_MALWARE_SCANNING_DAILY_AND_ON_DEMAND,
-		getTitle: () => i18n.translate( 'Daily and On-demand Malware Scanning' ),
+		getTitle: () => i18n.translate( 'Daily and on-demand malware scanning' ),
 		getDescription: () =>
 			i18n.translate(
 				'Automated security scanning with the option to run complete site scans at any time.'
 			),
 	},
 	[ constants.FEATURE_ONE_CLICK_THREAT_RESOLUTION ]: {
 		getSlug: () => constants.FEATURE_ONE_CLICK_THREAT_RESOLUTION,
-		getTitle: () => i18n.translate( 'One-click Threat Resolution' ),
+		getTitle: () => i18n.translate( 'One-click threat resolution' ),
 		getDescription: () =>
 			i18n.translate( 'Repair any security issues found on your site with just a single click.' ),
 	},
 	[ constants.FEATURE_AUTOMATIC_SECURITY_FIXES ]: {
 		getSlug: () => constants.FEATURE_AUTOMATIC_SECURITY_FIXES,
 		getTitle: () =>
-			i18n.translate( '{{strong}}Automatic{{/strong}} Security Fixes', {
+			i18n.translate( '{{strong}}Automatic{{/strong}} security fixes', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.